shared/chat/conversation/info-panel/header.js

```diff
@@ -1,57 +1,115 @@
 // @flow
 import * as React from 'react'
-import {Avatar, Box, ClickableBox, Text} from '../../../common-adapters'
-import {globalMargins, globalStyles, isMobile} from '../../../styles'
+import {Avatar, Box, ClickableBox, Icon, Text} from '../../../common-adapters'
+import {glamorous, globalMargins, globalStyles, isMobile} from '../../../styles'
 
 type SmallProps = {
   teamname: string,
   participantCount: number,
   onClick: () => void,
+  onClickGear: (?Element) => void,
 }
 
-const SmallTeamHeader = ({teamname, participantCount, onClick}: SmallProps) => (
-  <ClickableBox
-    style={{
-      ...globalStyles.flexBoxRow,
-      alignItems: 'center',
-      marginLeft: globalMargins.small,
-      marginTop: globalMargins.small,
-    }}
-    onClick={onClick}
-  >
-    <Avatar size={isMobile ? 48 : 32} teamname={teamname} isTeam={true} />
-    <Box style={{...globalStyles.flexBoxColumn, flex: 1, marginLeft: globalMargins.small}}>
-      <Text type="BodySemibold">{teamname}</Text>
-      <Box style={globalStyles.flexBoxRow}>
-        <Text type="BodySmall">
-          {participantCount.toString() + ' member' + (participantCount !== 1 ? 's' : '')}
-        </Text>
+const gearIconSize = isMobile ? 24 : 16
+
+const SmallTeamHeader = ({teamname, participantCount, onClick, onClickGear}: SmallProps) => {
+  const _onClick = (evt: SyntheticEvent<Element>) => {
+    if (!evt.defaultPrevented) {
+      onClick()
+    }
+  }
+  const _onClickGear = (evt: SyntheticEvent<Element>) => {
+    evt.preventDefault()
+    if (!isMobile) {
+      onClickGear(evt.currentTarget)
+    } else {
+      onClickGear()
+    }
+  }
+  return (
+    <ClickableBox
+      style={{
+        ...globalStyles.flexBoxRow,
+        alignItems: 'center',
+        marginLeft: globalMargins.small,
+      }}
+      // $FlowIssue with ClickableBox
```

Comment:
Seems to work to change `ClickableBox` (and anything else) to take `EventTarget<Element>`